mineinabyss-features/src/main/kotlin/com/mineinabyss/helpers/HelperFunctions.kt

```diff
@@ -53,4 +55,33 @@ fun Player.updateBalance() {
     }
 }
 
+fun Player.bossbarCompass(loc: Location, bar: BossBar) {
+    val player = player ?: return
 
+    val dir = loc.clone().subtract(player.location).toVector()
```

Comment:
this will cause funky behaviour if the two locations are in different worlds, I'd add a check for that (perhaps a custom `?` icon")

Reply:
If theyre not in the same world, itll show the :arrow_null: is the idea. Didn't consider different worlds tho so ill fix that